### PR TITLE
Move the SDK build to arcade

### PR DESCRIPTION
**Pull request by livarcocc**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -2,22 +2,30 @@
...
 
-  <Import Project="Sdk.props" Sdk="RoslynTools.RepoToolset" />
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <PropertyGroup>
     <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <DebugType>embedded</DebugType>
+    <DebugSymbols>true</DebugSymbols>
```
Comment (nguerrera) [inline Directory.Build.props]: Is this supposed to be always embedded? @tmat

File: Directory.Build.props [hunk 2]
```diff
@@ -2,22 +2,30 @@
...
+
+    <!--
+      Tools and packages produced by this repository support infrastructure and are not shipping on NuGet or via any other official channel.
+    -->
+    <IsShipping>true</IsShipping>
```
Comment (nguerrera) [inline Directory.Build.props]: I don't understand this comment vs. IsShipping=true.

File: Directory.Build.props [hunk 3]
```diff
@@ -2,22 +2,30 @@
...
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- 
+      'NetFxTfm' is the standard desktop Target Framework Moniker which this repo's packages are targeting
+      ie.  Place 'NetFxTfm' in the 'TargetFramework' property of a csproj like <TargetFrameworks>$(NetFxTfm);netcoreapp2.0</TargetFrameworks> 
+    -->
+    <NetFxTfm>net461</NetFxTfm>
```
Comment (nguerrera) [inline Directory.Build.props]: This is used by arcade somehow?

Comment (nguerrera) [reply Directory.Build.props]: I'm not seeing it in the diff anywhere else. Comment suggests we should be putting this in the project files.

File: Directory.Build.props (hunk 2 above)
Comment (livarcocc) [reply Directory.Build.props]: Yes, on the other thread, we agreed to keep IsShipping to true and to come up/set another property to indicate that the packages produced here are not going to NuGet, which is why the comment is there. 

Full disclosure, the props files, I took from @chcosta who started on the process of moving the SDK to arcade. But I think this comment is still relevant.

Comment (nguerrera) [reply Directory.Build.props]: I agree with IsShipping=true, but the comment seems to suggest the opposite. This repository ddoes more than support infrastructure.

Comment (livarcocc) [reply Directory.Build.props]: Sounds good. I will replace the comment with an issue to figure out this separate property to determine the nupkgs from here shuold not go to nuget.org. Sounds good?

File: Directory.Build.props (hunk 3 above)
Comment (livarcocc) [reply Directory.Build.props]: I will follow up with Chris on this.

**Pull request merged**